charts/openmetadata/templates/_validate-values.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "check_provider_value" }}
```

Comment:
Can you please update the filename to `validate-values.tpl` !

It seems partial files are do not support invocation of functions defined for Helm!

Reply:
updated the filename to validate-values.tpl